trustymail/cli.py

```diff
@@ -37,10 +37,10 @@
 """
 from trustymail import __version__
```

Comment:
While you're shuffling imports around, I would move this one to before the `from trustymail import trustymail` line below.

Reply:
Good idea.  I'll make the change.